.github/workflows/ci-cd.yml

```diff
@@ -69,3 +71,37 @@ jobs:
           SOURCE_DIR: ./umd
           DEST_DIR: sdk
           ARGS: --acl public-read --follow-symlinks --cache-control max-age=31536000,public
+
+
+      - name: SonarQube Scan (Push)
+        if: github.event_name == 'push'
+        uses: SonarSource/sonarcloud-github-action@v1.6
```

Comment:
SonarSource/sonarcloud-github-action is on 1.8, we can consider an update too

Reply:
Thanks! I updated it!